guides/v3.4.0/models/index.md

```diff
@@ -1,5 +1,19 @@
-Models are objects that represent the underlying data that your
-application presents to the user. Different apps will have very
+Ember developers have great options for how they handle data from back end APIs. 
+Ember itself works with any type of back end: REST, JSON API, GraphQL, or anything else.
+
+Many developers choose to use Ember Data, a powerful set of tools for formatting requests, serializing responses, and efficiently managing a local cache of data. 
+The Ember Data library is included by default in Ember apps. 
+Alternately, some developers write all their own code to handle API requests, using native JavaScript methods like [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) or third-party libraries.
+Many apps use a combination of approaches.
+
+This section of the Guides describes the essential features of Ember Data. To learn about other ways to handle data in an app, check out [Ember Observer](https://www.emberobserver.com/) for addons or search for community-made tutorials.
+
+## What are Ember Data models?
+
+In Ember Data, models are objects that represent the underlying data that your
+application presents to the user. 
```

Comment:
I am unsure what the best language here is, but here is how we use various terms on the `ember-data` team:

- `Model` => the `class` definition containing schema information about what `attributes` and `relationships` are valid for a particular JSON API `resource` whose `type` matches the `modelName` (the filepath).
- `record` => an instance of a `Model` for presenting the data of a single `resource` to the UI